en/faq/cloudflare.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Nuxt with with Cloudflare
+description: What needs to be considered when using Nuxt with Cloudflare?
+---
+
+# What needs to be considered when using Nuxt with Cloudflare?
+
+Nuxt can - in most cases - work with additional content (not generated / created by Nuxt). But sometimes additional content can cause problems with Nuxt - especially Cloudflare's "Minification and Security Options".
+
+Accordingly, you should make sure that the following options are unchecked / disabled in Cloudflare:
```

Comment:
```suggestion
Accordingly, you should make sure that the following options are unchecked / disabled in Cloudflare. Otherwise, unnecessary re-rendering or hydration errors could impact your production application.
```